2020-03-18 17:42:03,439 - my_interface_test - [test_01_register.py-->line:29] - INFO: 用例正确格式的数据注册成功断言成功
2020-03-18 17:42:04,244 - my_interface_test - [test_01_register.py-->line:29] - INFO: 用例密码为空断言成功
2020-03-18 17:42:05,073 - my_interface_test - [test_01_register.py-->line:29] - INFO: 用例密码少于8位断言成功
2020-03-18 17:42:05,867 - my_interface_test - [test_01_register.py-->line:29] - INFO: 用例密码大于16位断言成功
2020-03-18 17:42:06,323 - my_interface_test - [test_01_register.py-->line:29] - INFO: 用例手机号为空断言成功
2020-03-18 17:42:06,779 - my_interface_test - [test_01_register.py-->line:29] - INFO: 用例手机号密码均为空断言成功
2020-03-18 17:42:07,226 - my_interface_test - [test_01_register.py-->line:29] - INFO: 用例手机号输入为空格断言成功
2020-03-18 17:42:07,572 - my_interface_test - [test_01_register.py-->line:25] - ERROR: 用例手机号和密码均输入为空格断言失败
2020-03-18 17:42:13,264 - my_interface_test - [test_01_register.py-->line:25] - ERROR: 用例手机号和密码均输入为空格断言失败
2020-03-18 17:42:18,920 - my_interface_test - [test_01_register.py-->line:25] - ERROR: 用例手机号和密码均输入为空格断言失败
2020-03-18 17:42:19,701 - my_interface_test - [test_02_login.py-->line:39] - INFO: 用例手机密码均为空断言成功
2020-03-18 17:42:20,186 - my_interface_test - [test_02_login.py-->line:35] - ERROR: 用例手机密码正常输入断言失败
2020-03-18 17:42:25,677 - my_interface_test - [test_02_login.py-->line:35] - ERROR: 用例手机密码正常输入断言失败
2020-03-18 17:42:31,171 - my_interface_test - [test_02_login.py-->line:35] - ERROR: 用例手机密码正常输入断言失败
2020-03-18 17:42:31,640 - my_interface_test - [test_02_login.py-->line:39] - INFO: 用例密码为空断言成功
2020-03-18 17:42:32,062 - my_interface_test - [test_02_login.py-->line:39] - INFO: 用例手机号少于8位断言成功
2020-03-18 17:42:32,820 - my_interface_test - [test_03_recharge.py-->line:53] - INFO: 用例不登录充值断言成功
2020-03-18 17:42:56,184 - my_interface_test - [test_03_recharge.py-->line:49] - ERROR: 用例投资人id为空断言失败
2020-03-18 17:43:01,712 - my_interface_test - [test_03_recharge.py-->line:49] - ERROR: 用例投资人id为空断言失败
2020-03-18 17:43:07,192 - my_interface_test - [test_03_recharge.py-->line:49] - ERROR: 用例投资人id为空断言失败
2020-03-18 17:43:07,653 - my_interface_test - [test_03_recharge.py-->line:49] - ERROR: 用例投资人id为空格断言失败
2020-03-18 17:43:13,121 - my_interface_test - [test_03_recharge.py-->line:49] - ERROR: 用例投资人id为空格断言失败
2020-03-18 17:43:18,581 - my_interface_test - [test_03_recharge.py-->line:49] - ERROR: 用例投资人id为空格断言失败
2020-03-18 17:43:19,003 - my_interface_test - [test_03_recharge.py-->line:49] - ERROR: 用例投资数量为负数断言失败
2020-03-18 17:43:24,487 - my_interface_test - [test_03_recharge.py-->line:49] - ERROR: 用例投资数量为负数断言失败
2020-03-18 17:43:29,990 - my_interface_test - [test_03_recharge.py-->line:49] - ERROR: 用例投资数量为负数断言失败
2020-03-18 17:43:30,427 - my_interface_test - [test_03_recharge.py-->line:49] - ERROR: 用例投资数量为小数断言失败
2020-03-18 17:43:35,933 - my_interface_test - [test_03_recharge.py-->line:49] - ERROR: 用例投资数量为小数断言失败
2020-03-18 17:43:41,440 - my_interface_test - [test_03_recharge.py-->line:49] - ERROR: 用例投资数量为小数断言失败
2020-03-18 17:43:41,929 - my_interface_test - [test_03_recharge.py-->line:49] - ERROR: 用例投资数量为非字数断言失败
2020-03-18 17:43:47,395 - my_interface_test - [test_03_recharge.py-->line:49] - ERROR: 用例投资数量为非字数断言失败
2020-03-18 17:43:52,880 - my_interface_test - [test_03_recharge.py-->line:49] - ERROR: 用例投资数量为非字数断言失败
2020-03-18 17:43:53,332 - my_interface_test - [test_03_recharge.py-->line:49] - ERROR: 用例投资数量为空断言失败
2020-03-18 17:43:58,799 - my_interface_test - [test_03_recharge.py-->line:49] - ERROR: 用例投资数量为空断言失败
2020-03-18 17:44:04,314 - my_interface_test - [test_03_recharge.py-->line:49] - ERROR: 用例投资数量为空断言失败
2020-03-18 17:44:04,767 - my_interface_test - [test_03_recharge.py-->line:49] - ERROR: 用例投资数量为0断言失败
2020-03-18 17:44:10,549 - my_interface_test - [test_03_recharge.py-->line:49] - ERROR: 用例投资数量为0断言失败
2020-03-18 17:44:16,308 - my_interface_test - [test_03_recharge.py-->line:49] - ERROR: 用例投资数量为0断言失败
2020-03-18 17:44:17,320 - my_interface_test - [test_04_add.py-->line:43] - INFO: 用例未登录增加项目断言成功
2020-03-18 17:44:41,219 - my_interface_test - [test_04_add.py-->line:39] - ERROR: 用例用户id为空断言失败
2020-03-18 17:44:46,952 - my_interface_test - [test_04_add.py-->line:39] - ERROR: 用例用户id为空断言失败
2020-03-18 17:44:52,668 - my_interface_test - [test_04_add.py-->line:39] - ERROR: 用例用户id为空断言失败
2020-03-18 17:44:53,447 - my_interface_test - [test_04_add.py-->line:39] - ERROR: 用例未注册的用户id断言失败
2020-03-18 17:44:59,262 - my_interface_test - [test_04_add.py-->line:39] - ERROR: 用例未注册的用户id断言失败
2020-03-18 17:45:05,085 - my_interface_test - [test_04_add.py-->line:39] - ERROR: 用例未注册的用户id断言失败
2020-03-18 17:45:05,772 - my_interface_test - [test_04_add.py-->line:39] - ERROR: 用例标题为空断言失败
2020-03-18 17:45:11,493 - my_interface_test - [test_04_add.py-->line:39] - ERROR: 用例标题为空断言失败
2020-03-18 17:45:17,248 - my_interface_test - [test_04_add.py-->line:39] - ERROR: 用例标题为空断言失败
2020-03-18 17:45:17,938 - my_interface_test - [test_04_add.py-->line:39] - ERROR: 用例借款数量为0断言失败
2020-03-18 17:45:23,907 - my_interface_test - [test_04_add.py-->line:39] - ERROR: 用例借款数量为0断言失败
2020-03-18 17:45:29,646 - my_interface_test - [test_04_add.py-->line:39] - ERROR: 用例借款数量为0断言失败
2020-03-18 17:45:30,358 - my_interface_test - [test_04_add.py-->line:39] - ERROR: 用例借款数量为负数断言失败
2020-03-18 17:45:36,084 - my_interface_test - [test_04_add.py-->line:39] - ERROR: 用例借款数量为负数断言失败
2020-03-18 17:45:41,819 - my_interface_test - [test_04_add.py-->line:39] - ERROR: 用例借款数量为负数断言失败
2020-03-18 17:45:42,520 - my_interface_test - [test_04_add.py-->line:39] - ERROR: 用例借款期限为负数断言失败
2020-03-18 17:45:48,239 - my_interface_test - [test_04_add.py-->line:39] - ERROR: 用例借款期限为负数断言失败
2020-03-18 17:45:53,957 - my_interface_test - [test_04_add.py-->line:39] - ERROR: 用例借款期限为负数断言失败
2020-03-18 17:45:54,661 - my_interface_test - [test_04_add.py-->line:39] - ERROR: 用例借款期限小于1个月断言失败
2020-03-18 17:46:00,414 - my_interface_test - [test_04_add.py-->line:39] - ERROR: 用例借款期限小于1个月断言失败
2020-03-18 17:46:06,172 - my_interface_test - [test_04_add.py-->line:39] - ERROR: 用例借款期限小于1个月断言失败
2020-03-18 17:46:06,891 - my_interface_test - [test_04_add.py-->line:39] - ERROR: 用例借款期限大于36个月断言失败
2020-03-18 17:46:12,871 - my_interface_test - [test_04_add.py-->line:39] - ERROR: 用例借款期限大于36个月断言失败
2020-03-18 17:46:18,873 - my_interface_test - [test_04_add.py-->line:39] - ERROR: 用例借款期限大于36个月断言失败
